test/integration/minishift.go

```diff
@@ -105,6 +110,13 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 }
 
 func (m *Minishift) getOpenShiftUrl() string {
-	cmdOut, _, _ := m.runner.RunCommand("console --url")
+	cmdOut, _, _ := m.runner.RunCommand(ConsoleURL)
 	return strings.TrimRight(cmdOut, "\n")
 }
+
+func (m *Minishift) getRoutingUrl() string {
```

Comment:
Done. As per the current implementation i can get the routing url  just be executing this ```get route -o=custom-columns=URL:.spec.host``` if there is only one application is deployed. Will check how it will be done in a better way.